qhub_ops/template/{{ cookiecutter.repo_directory }}/image/Dockerfile.jupyterlab

```diff
@@ -11,20 +17,24 @@ RUN /opt/scripts/install-conda.sh
 
 ENV PATH="/opt/conda/bin:$PATH"
 
-# ========== jupyterlab install ============
-COPY jupyterlab /opt/jupyterlab
+CMD [ "/bin/bash" ]
 
-RUN /opt/scripts/install-apt.sh                  /opt/jupyterlab/apt.txt \
-    && /opt/scripts/install-conda-environment.sh /opt/jupyterlab/environment.yaml \
-    && /opt/jupyterlab/postBuild
-
-# ========== regular user ===========
 ARG NB_USER="jovyan"
-# We assume only one user per container, so make it unique UID matches
-# GID
+# We assume only one user per container, so make it unique
+# UID matches GID
 ARG NB_UID="1000"
 
-RUN groupadd --gid ${NB_UID} ${NB_USER} && \
+ENV SHELL=/bin/bash \
```

Comment:
In my opinion setting `SHELL=/bin/bash` is all this PR needs since conda doesn't work well with `sh`. Did this not work?

Reply:
This doesn't initialize conda for bash shell.